strictequal.js

```diff
@@ -0,0 +1,7 @@
+const strictEqual = (a, b) => {
+    if (typeof a == typeof b && a == b) {
+        return true;
```

Comment:
you actually dont need the return true or false that function will automatically return either answer.